src/auto-evo/steps/GenerateMiche.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace AutoEvo;
+
+public class GenerateMiche : IRunStep
+{
+    public Patch Patch;
+    public SimulationCache Cache;
+
+    private static readonly Compound Glucose = SimulationParameters.Instance.GetCompound("glucose");
```

Comment:
These being static is a bit iffy to me as that keeps state for the whole program. I've much preferred using non-static references to the registry types as that still reduces the lookups a lot but doesn't force keep the references around for the whole program runtime, which could be a problem in some modding contexts (that's basically all I got, but I'd like to keep the future flexibility and avoid static references to registry stuff).

Reply:
Also doing the references in the constructor would allow eliminating duplicate property reads (`SimulationParameters.Instance` is a property that I assume the compiler has to re-read for each of these field initializers).

Reply:
So should it just be const?

Reply:
It cannot be const as this needs to be looked up during runtime (in hindsight it probably would have been better if Compound was an enum but we don't live in that timeline sadly). So just remove `static` and leave it be readonly and fetch all of these compounds in the constructor.